### DISCUSSION TITLE
Color-tagged text has issue in garbled speech

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15368] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![20241214180056](https://github.com/user-attachments/assets/4df727d8-ecff-45dd-9de7-491a682482ef)

### Reproduction steps

send  `There's something alien growing in ‖color:gui.Blue‖Gunnery Compartment‖color:end‖.` near max communication range

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested a fix for this.
Closing discussion as resolved.
